### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by 4xmplme**
- [DEMO LINK](https://4xmplme.github.io/layout_stars/)
- [TEST REPORT LINK](https://4xmplme.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-of-type) {
+  margin-right: 4px;
+}
+
+/* How can I optimize this selectors to don't break DRY rule? */
```
Comment (vlad-demchuk) [inline src/style.css:21]: looks good, just remove the comments

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-of-type) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(1),
```
Comment (OleziO) [inline src/style.css:21]: Make all code in one style. Just replace it with `-n + 1`

**Review by OleziO — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 3]
```diff
@@ -8,6 +8,47 @@
...
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
```
Comment (DarkMistyRoom) [inline src/index.html:18]: Add empty lines between multiline sibling blocks of HTML

**Review by DarkMistyRoom — APPROVED** [review]
Good job!

**Pull request closed without merging**